tests/Unit/Support/Filters/WildcardFilterTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use Worksome\Envy\Support\Filters\WildcardFilter;
+
+it('matches based on the given wildcard character', function (string $variable, string $comparison, bool $matches) {
+    $filter = new WildcardFilter($variable, '%');
+
+    expect($filter->environmentVariableMatches($comparison))->toBe($matches);
+})->with([
+    ['FOO_%', 'FOO_BAR', true],
+    ['FOO_%', 'FOO_BAZ', true],
+    ['FOO_%', 'FOO_', false],
+    ['FOO_%', 'BAZ_FOO', false],
+    ['FOO_%', 'BAZ', false],
+    ['FOO%BAR', 'FOO_BAR', true],
```

Comment:
Could you add one here for checking if `FOO_BAR_BAR` is matched?

Reply:
Added 👍